aws/setStackContext.sh

```diff
@@ -162,6 +162,15 @@ ALTERNATIVE_CONFIGS=$(findFiles \
     "${CF_DIR}/${LEVEL_PREFIX}${DEPLOYMENT_UNIT_PREFIX}${DEPLOYMENT_UNIT_SUBSET_PREFIX}${REGION_PREFIX}*-config.json" \
     "${CF_DIR}/${LEVEL_PREFIX}${DEPLOYMENT_UNIT_PREFIX}${DEPLOYMENT_UNIT_SUBSET_PREFIX}${ACCOUNT_PREFIX}${REGION_PREFIX}*-config.json" )
 
+CLI="${LEVEL_PREFIX}${DEPLOYMENT_UNIT_PREFIX}${DEPLOYMENT_UNIT_SUBSET_PREFIX}${REGION_PREFIX}cli.json"
```

Comment:
As cli is new, there's no need for this conditional logic - it should always include the account id